README.md

```diff
@@ -344,54 +344,73 @@ $ espercli group list [OPTIONS]
 
 ##### Example
 ```sh
-$ espercli group list -n 5G
-Number of Groups: 1
-
-ID                                    NAME      DEVICE COUNT
-2e5efca2-7776-442e-a5ef-c2758d4a45a3  5G                   2
+$ espercli group list -n TestBB 
+Number of Groups: 2
+ID                                    NAME      DEVICE COUNT  PARENT ID
+bb6be630-a2fc-4b57-bafa-fdda92433684  TestBB               0  685484fa-eb6a-4ef9-a0f1-63f6febf7ce3
+5fd4e150-7a91-4d73-a379-cac0795bd949  TestBB               0  bb6be630-a2fc-4b57-bafa-fdda92433684
 ```
 
 #### 2. show
-Show the details of the group. Here, `group-name` is required to show group information. 
+Shows the details of the group. Here, `group-name` is required and will return the information of the first group from the response list with the given name. Since nested groups allow the same name in different hierarchy levels, `group-name` and `--groupid` or `-id` option can be given together to show the corresponding group information.
 Use the `--active` or `-a` flag to mark this group as the active group. This will allow you to call further group commands without specifying the group.
 ```sh
 $ espercli group show [OPTIONS] [group-name]
 ```
 ##### Options
 | Name, shorthand | Default| Description|
 | -------------   |:------:|:----------|
-| --active, -a    |        | Set device as active for further device specific commands |
+| --groupid, -id  |        | Group id  |
+| --active, -a    |        | Set group as active for further device specific commands |
```

Comment:
Set group as active for further device specific commands => Set group as active for further group specific commands, right?